automatic/Firefox/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,9 @@
+$unfile = "Mozilla Firefox\uninstall\helper.exe"
+
+if (Test-Path "${Env:ProgramFiles(x86)}\$unfile"){
+   $unpath = "${Env:ProgramFiles(x86)}\$unfile"
+}
+else {
+   $unpath = "${Env:ProgramFiles}\$unfile"
+}
```

Comment:
Our preference for packages is to use the registry to get the uninstall location.